RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate methods/callJitsi to typescript

## Proposed changes

Migrate methods/callJitsi to typescript

## Issue(s)	


## How to test or reproduce
Jitsi without server:
* Create a Jitsi meet with a random url, for example https://jitsi.rocket.chat/test-call, Send as a link on a chat, tap this link and then a meeting room should open normally.

Jitsi with server: 
* open a chat -> tap on top menu to open the action screen:
* try to make a voice call;
* try to make a video call;

everything needs works normally

## Screenshots

## Types of changes




## Checklist



## Further comments
